check50/__main__.py

```diff
@@ -278,17 +284,25 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
     if args.dev:
         args.offline = True
-        args.verbose = True
+        args.verbose = "info"
```

Comment:
Should that override `args.verbose` only if `args.verbose` isn't passed? What if the user wants another verbosity level in dev mode?

Reply:
You're right, done!